trackers/javascript-tracker/tracker.config.ts

```diff
@@ -28,23 +28,26 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-export const performanceTiming = true;
+/* By default included plugins in sp.js */
+export const adTracking = true;
+export const clientHints = true;
+export const enhancedConsent = true;
+export const errorTracking = true;
+export const formTracking = true;
 export const gaCookies = true;
 export const geolocation = true;
-export const optimizelyX = true;
-export const clientHints = true;
-export const consent = true;
 export const linkClickTracking = true;
-export const formTracking = true;
-export const errorTracking = true;
-export const timezone = true;
-export const ecommerce = true;
-export const enhancedEcommerce = true;
-export const adTracking = true;
+export const performanceTiming = true;
 export const siteTracking = true;
-export const snowplowEcommerceTracking = false;
-export const optimizely = false;
+export const snowplowEcommerceTracking = true;
+export const timezone = true;
+
+/* By default excluded plugins in sp.js */
 export const browserFeatures = false;
+export const consent = false;
+export const ecommerce = false;
+export const enhancedEcommerce = false;
 export const mediaTracking = false;
+export const optimizely = false;
+export const optimizelyX = false;
 export const youtubeTracking = false;
```

Comment:
I like this separation. I wonder if it's worth making a "Included", "Excluded", "Deprecated" sections for anyone who does stumble into this file.

I'd say:
browserFeatures, consent, ecommerce, enhancedEcommerce and optimisely (X can stay undeprecated but non-default) are all (and if not should be) deprecated now.

Reply:
Done